crates/markdown_preview/src/markdown_preview_view.rs

```diff
@@ -515,19 +517,21 @@ impl Render for MarkdownPreviewView {
     fn render(&mut self, _window: &mut Window, cx: &mut Context<Self>) -> impl IntoElement {
         let buffer_size = ThemeSettings::get_global(cx).buffer_font_size(cx);
         let buffer_line_height = ThemeSettings::get_global(cx).buffer_line_height;
-        v_flex()
-            .id("MarkdownPreview")
-            .key_context("MarkdownPreview")
-            .track_focus(&self.focus_handle(cx))
-            .size_full()
-            .bg(cx.theme().colors().editor_background)
-            .p_4()
-            .text_size(buffer_size)
-            .line_height(relative(buffer_line_height.value()))
-            .child(
-                div()
-                    .flex_grow()
-                    .map(|this| this.child(list(self.list_state.clone()).size_full())),
-            )
+        image_cache(self.image_cache.clone()).child(
+            v_flex()
+                .id("MarkdownPreview")
+                .key_context("MarkdownPreview")
+                .track_focus(&self.focus_handle(cx))
```

Comment:
I'd think here if we have a `.image_cache` method for Div like this `key_context` or `track_focus` method, then we can reduce an indent depth.

How do you think about this?